### DISCUSSION TITLE
why 2 links aren't working on terminology page

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#228] [Help]
The [terminology page](https://github.com/UNC-Project-Open-AAC/OS-DPI/wiki/Terminology) is giving me trouble. 

@gbishop Why are the links for Modal Dialog and Tab Control not playing nice with the format [Modal Dialog](../Modal Dialog) ?

**Comment by gbishop** [comment]
You needed to replace the space in the link with a dash. I fixed it.
